cmd/auth/main.go

```diff
@@ -47,6 +49,9 @@ const (
 	defServerCert    = ""
 	defServerKey     = ""
 	defJaegerURL     = ""
+	ketoContainer    = "mainflux-keto"
```

Comment:
Should this be configurable?

Reply:
No, it should not be configurable. I've only declared it because I am using `mainflux-keto` in multiple places across `cmd/auth/main.go`. Should `ketoContainer` constant usage be replaced with plain string?

Reply:
It should be configurable, container names (and URLs) can change easily. From what I see you are dialing this URL from the init function - changing docker-compose, renaming containers, using K8s - or whatever similar - can change container name (and thus hostname, or URL in a private network) and then your code will stop working, because you hard-coded it.